src/Authoring/WinRT.SourceGenerator/WinRTRules.cs

```diff
@@ -124,12 +124,6 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
             "Array signature found with multi-dimensional array, which is not a valid Windows Runtime type",
             "Method '{0}' has a multi-dimensional array of type '{1}' in its signature. Arrays in Windows Runtime method signatures must be one dimensional.");
 
```

Comment:
Is this not needed?

Reply:
No I used this at one point, but have switched to a more general "not valid type" rule, and this one has been unused for a while.